src/components/com_tjcertificate/administrator/views/trainingrecord/tmpl/edit.php

```diff
@@ -51,15 +51,15 @@
 		}
 		?>
 		<div class="form-horizontal">
-
-		<?php echo HTMLHelper::_('bootstrap.addTab', 'myTab', 'general', Text::_('COM_TJCERTIFICATE_TITLE_CERTIFICATE')); ?>
+		<?php
+			echo HTMLHelper::_('bootstrap.startTabSet', 'myTab', array('active' => 'general'));
+			echo HTMLHelper::_('bootstrap.addTab', 'myTab', 'general', Text::_('COM_TJCERTIFICATE_TITLE_CERTIFICATE')); ?>
```

Comment:
Now you have added two new sets of code. @divyachaudhari 
Is there anything you fixed here? else keep it is as it was earlier

Reply:
For the tab, The warning were dsiplayed and the "Issues Certificate" tab was not displayed due to the warnings. This code has removed the warning and also displaying the tab "Issued certificate"